src/playlist-review/playlist-review-store.ts

```diff
@@ -13,12 +13,13 @@ export class PlaylistReviewStore {
     @observable
     currentAudio: HTMLAudioElement | undefined;
 
-    constructor(private readonly spotifyApi: SpotifyApi, private readonly selectedGenres: string[], private readonly selectedArtists: string[]) {
+    constructor(private readonly spotifyApi: SpotifyApi, private readonly selectedGenres: string[],
+                private readonly selectedArtists: string[], private readonly playlistLimit: number) {
         this.init();
     }
 
     private init() {
-        this.spotifyApi?.getRecommendations(this.selectedGenres, this.selectedArtists)
+        this.spotifyApi?.getRecommendations(this.playlistLimit, this.selectedGenres, this.selectedArtists)
```

Comment:
See my comment above about params order

Reply:
If I move it to the end I have another default value "retry" - what should I do in this situation?